.scalafix.conf

```diff
@@ -1,10 +1,13 @@
 rules = [
   OrganizeImports,
   ExplicitResultTypes,
+  RemoveUnused
```

Comment:
Added back the `RemoveUnused` rule to remove unused private and local values. Left imports to the OrganizeImports rules.